frontend/src/routes/Video.tsx

```diff
@@ -250,6 +262,40 @@ const VideoTitle: React.FC<VideoTitleProps> = ({ title }) => (
     }} />
 );
 
+type EmbedCodeProps = {
+    event: {
+        opencastId: string;
+    };
+};
+
+const EmbedCode: React.FC<EmbedCodeProps> = ({ event: { opencastId: id } }) => {
+    const { t } = useTranslation();
+
+    const modal = useRef<ModalHandle>(null);
+
+    const embedCode = `<iframe ${[
+        `src="${CONFIG.opencast.presentationNode}/play/${id}"`,
+        "allowfullscreen",
+        `style="${[
+            "border: none;",
+            "width: 100%;",
+            "aspect-ratio: 16/9;",
+        ].join(" ")}`,
```

Comment:
```suggestion
        ].join(" ")}"`,
```

That was missing. But see other comment, I think I would prefer this as one big template literal.